src/main/java/luke/Main.java

```diff
@@ -0,0 +1,149 @@
+                    Event newEvent = createEvent(input);
+                    taskList.writeTasks(tlFile, newEvent);
+                } catch (EmptyEventException e) {
+                    System.out.printf("Luke:%s\nYou:\n", e.getMessage());
+                } catch (InvalidEventException e) {
+                    System.out.printf("Luke:%s\nYou:\n", e.getMessage());
+                }
+            } else if (input.equals("list")) {
+                if (taskList.getSize() < 1) {
+                    System.out.printf("Luke:\n\tYou don't have any tasks in your list :(\nYou:\n");
+                } else {
+                    String todoSummary = "Luke:\n\tHere are the tasks in your list.";
+                    for (int i = 0; i < taskList.getSize(); i++) {
+                        Task current = taskList.getTask(i);
+                        todoSummary += String.format("\n\t%d.%s", i + 1, current);
+                    }
+                    System.out.printf("%s\nYou:\n", todoSummary);
+                }
+            } else if (Pattern.matches("^(done) *[0-9]*$", input)) {
+               try {
+                   taskList.completeTask(input);
+               } catch (InvalidDoneException e) {
+                   System.out.printf("Luke:%s\nYou:\n", e.getMessage());
+               } catch (DoneIndexOutofboundsException e) {
+                   System.out.printf("Luke:%s\nYou:\n", e.getMessage());
+               }
+            } else if (Pattern.matches("^(delete) *[0-9]*$", input)) {
+                try {
+                    taskList.deleteTask(input, tlFile);
+                } catch (InvalidDeleteException e) {
+                    System.out.printf("Luke:%s\nYou:\n", e.getMessage());
+                } catch (DeleteIndexOutofboundsException e) {
+                    System.out.printf("Luke:%s\nYou:\n", e.getMessage());
+                }
+            } else if (Pattern.matches("^(find) *.*$", input)) {
+                input.replaceAll("find ", "");
+                taskList.findTask(input);
+            } else if (input.equals("bye")) {
+                System.out.println("Luke:\n\tOh, are you leaving? Hope to see you soon!");
+                break;
+            } else {
+                System.out.printf("Luke:\n\tSorry I do not understand :( Please try another command.\nYou:\n");
+            }
+        }
+    }
+
+    @Override
+    public void start(Stage stage) {
+        Label helloWorld = new Label("Hello World!"); // Creating a new Label control
+        Scene scene = new Scene(helloWorld); // Setting the scene to be our Label
+
+        stage.setScene(scene); // Setting the stage to show our screen
+        stage.show(); // Render the stage.
+    }
+
+    /**
+     * Creates a task.Todo object with the given input.
+     *
+     * @param input description of the task.Todo object
+     * @return the todo object with the given input
+     */
+    private static Todo createTodo(String input) throws EmptyTodoException {
```

Comment:
The `Main` class seems to be doing too many things. I would suggest putting all these `createXXXX` functions in another class.

Reply:
Noted! Thanks for the detailed suggestion.